src/Controller/PetController.php

```diff
@@ -15,20 +16,15 @@ class PetController extends AbstractController
     /**
      * @Route("/new", name="new_pet")
      */
-    public function new(Request $request) : Response
+    public function new(Request $request, PetService $petService) : Response
```

Comment:
I would recommend injecting stuff into constructor. https://www.tomasvotruba.cz/blog/2018/04/23/how-to-slowly-turn-your-symfony-project-to-legacy-with-action-injection/